src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1048,7 +1048,8 @@ static bool isOrContainsErrorType(TypeSymbol typeSymbol)
                                                              ErrorCode.WRN_NullabilityMismatchInParameterTypeOnOverride,
                                                              location,
                                                              new FormattedSymbol(overridingParameter, SymbolDisplayFormat.ShortFormat));
-                                                     } :
+                                                     }
+                :
```

Comment:
This is just begging for some readability improvements.

Reply:
this is a formatter bug, right?

Reply:
Agree with Rikki that this looks like a bug at a glance. Is it a bug or is this the intended behavior? If it's a bug do we have something tracking it? Not blocking but want to make sure we track odd behaviors here.

Or if it's by design interested in the rationale

---
In reply to: [321930329](https://github.com/dotnet/roslyn/pull/38562#discussion_r321930329) [](ancestors = 321930329)

Reply:
This will need to be a follow-up bug. It won't let me change it.

Reply:
I'm just saying that it was close to inscrutable before the formatter got to it. ;)

Reply:
➡️ https://github.com/dotnet/roslyn/issues/38588